src/Conjure/Language/ParserC.hs

```diff
@@ -1,698 +1,20 @@
 {-# LANGUAGE RecordWildCards #-}
 
-module Conjure.Language.ParserC ( parseModel ) where
+module Conjure.Language.ParserC (   
```

Comment:
Note: this means we will be allowing operators in the param/solution files, which might make parsing a bit slower for them.

Reply:
New parser (probably mostly from new megaparsec) seems to be at least 25% faster so hopefully this is offset somewhat